geomstats/information_geometry/normal.py

```diff
@@ -105,7 +109,12 @@ def pdf(x):
                 Points at which to compute the probability density function.
             """
             pdf_at_x = [
-                gs.array(norm.pdf(x, loc=mean, scale=std))
+                gs.array(
```

Comment:
can't we code this as a function? looks reusable.

Reply:
Done.